hikari/impl/rest.py

```diff
@@ -737,6 +737,7 @@ async def _request(
                         await live_attributes.still_alive().global_rate_limit.acquire()
 
                     if trace_logging_enabled:
+                        start = time.monotonic()
```

Comment:
Should probably be moved back down where it was so that it doesnt take into account the time of stringifying the http message